### PR TITLE
Fix refactored code

### PR DESCRIPTION
**Pull request by dbobrenko**
## Changes
  - Fix organic scoring;
  - Fix LLM inference;
  - Add organic scoring env variables.

### CODE REVIEW

File: prompting/organic/organic_scoring_prompting.py [hunk 1]
```diff
@@ -226,37 +220,38 @@ async def _wait_for_completion(uid: int):
...
-        )
+
+        async with dendrite(wallet=settings.WALLET) as dend:
+            streams_responses = await dend(
+                axons=[settings.METAGRAPH.axons[uid] for uid in uids],
+                synapse=StreamPromptingSynapse(roles=sample["roles"], messages=sample["messages"]),
+                timeout=settings.ORGANIC_TIMEOUT,
+                deserialize=False,
+                streaming=True,
+            )
```
Comment (cassova) [inline prompting/organic/organic_scoring_prompting.py:240]: Can you explain to me why you changed this from `await` to `async with`?

File: prompting/organic/organic_scoring_prompting.py [hunk 2]
```diff
@@ -176,7 +173,7 @@ async def stream_miner_chunks(uid: int, chunks: AsyncGenerator):
             return_exceptions=True,
         )
 
-    async def _reuse_organic_response(self, sample: OrganicEntry) -> dict[int, SynapseStreamResult]:
+    async def _reuse_organic_response(self, sample: dict[str, Any]) -> dict[int, SynapseStreamResult]:
```
Comment (cassova) [inline prompting/organic/organic_scoring_prompting.py:176]: nit: if we can avoid using the type `Any`, the better.

File: prompting/organic/organic_scoring_prompting.py [hunk 3]
```diff
@@ -157,7 +154,7 @@ async def stream_miner_chunks(uid: int, chunks: AsyncGenerator):
                     elif isinstance(chunk, StreamPromptingSynapse):
                         synapse = chunk
                 except Exception as e:
-                    logger.error(f"[Organic] Error while streaming chunks: {e}")
+                    logger.exception(f"[Organic] Error while streaming chunks")
```
Comment (cassova) [inline prompting/organic/organic_scoring_prompting.py:157]: this is cool. just read that this automatically adds tracebook.  Note, you get the same effect with `logger.error(<message>, stack_info=True, exc_info=True`

**Review by cassova — APPROVED** [review]
minor q/a's

File: prompting/organic/organic_scoring_prompting.py (hunk 1 above)
Comment (dbobrenko) [reply prompting/organic/organic_scoring_prompting.py:240]: It works for both cases. Async with just ensures we are using proper asyncio loop, with the given context

File: prompting/organic/organic_scoring_prompting.py (hunk 2 above)
Comment (dbobrenko) [reply prompting/organic/organic_scoring_prompting.py]: Added dataclasses and more precise type hints. Changing `sample` to dataclass is some bigger change, which requires modifying organic-scoring repo

**Pull request merged**